api.js

```diff
@@ -1902,4 +1903,51 @@ API.prototype.ignoreApdex = function ignoreApdex() {
   transaction.ignoreApdex = true
 }
 
+/**
+ * Runs a function synchronously within a provided LLM custom attributes context and returns its return value.
+ *
+ * An example of setting a custom attribute:
+ *
+ *    newrelic.withLlmCustomAttributes({'llm.someAttribute': 'someValue'}, () => {
+ *      return;
+ *    })
+ * @param {Object} context LLM custom attributes context
+ * @param {Function} callback synchronous function called within the context
```

Comment:
this isn't true. it's just a function. the asynclocalstorage will bind any async operations within the callback accordingly